src/components/pages/template/edit/field-tree.tsx

```diff
@@ -38,22 +50,184 @@ export default function FieldTree() {
   const generateNodeProps = (row) => ({
     title: row.node.name,
     buttons: [
-      <IconButton
-        minW="auto"
-        aria-label="Edit Field"
-        variant="link"
-        onClick={() => setCurrentField(row.node)}
-        icon={<EditIcon />}
-      />,
-      <IconButton
-        colorScheme="red"
-        minW="auto"
-        aria-label="Delete Field"
-        variant="link"
-        onClick={() => removeFieldById(row.node.fieldId)}
-        icon={<DeleteIcon />}
-      />
-    ]
+      (row.node.type == "TEXT" || row.node.type == "TEXTAREA" || row.node.type == "RICHTEXT") && (
+        <Tooltip label="Text">
```

Comment:
For all the tooltip labels, instead of directly hard-coding the string,make use of `useTranslation()` hook and put the required value in the translations files.
Refer to [this](https://github.com/biodiv-platform/cca-ui/blob/0be8465e5d001d64381d1a0b218183086b32bf6e/src/components/@core/loading/index.tsx) component as an example.